test/boost/mutation_reader_test.cc

```diff
@@ -1068,7 +1069,7 @@ SEASTAR_TEST_CASE(test_fast_forwarding_combined_reader_is_consistent_with_slicin
                 mutation_reader::forwarding::yes));
         }
 
-        flat_mutation_reader rd = downgrade_to_v1(make_combined_reader(s, permit, std::move(readers),
+        auto rd = mutation_fragment_v1_stream(make_combined_reader(s, permit, std::move(readers),
```

Comment:
This tests should just be converted to v2, instead of moving it to use the converter. There is nothing v1 specific here AFAIK.

Reply:
Not resolved.